.github/workflows/heroku-deploy.yml

```diff
@@ -0,0 +1,67 @@
+---
+name: Deploy mocks on Heroku
+on:  # yamllint disable-line rule:truthy
+  # Deploy the mocks on Heroku on every push to master branch
+  push:
+    branches:
+      - master
+
+env:
+  HEROKU_APP_NAME: pets-consumer-contract-mock
```

Comment:
Todo in a future commit: make the `HEROKU_APP_NAME` dynamic, based on the GitHub repository name.

Reply:
Yep it is a good update